src/Felix/Laws.agda

```diff
@@ -274,7 +274,7 @@ record CartesianClosed {obj : Set o} ⦃ _ : Products obj ⦄
                        {q} ⦃ _ : Equivalent q _⇨′_ ⦄
                        ⦃ _ : R.Category _⇨′_ ⦄
                        ⦃ _ : R.Cartesian _⇨′_ ⦄
-                       ⦃ _ : R.CartesianClosed _⇨′_ ⦄
+                       ⦃ raw : R.CartesianClosed _⇨′_ ⦄
```

Comment:
Probably merits a more specific name, considering the two previous instance parameters.